README.md

```diff
@@ -2,29 +2,215 @@
+editable installation of his package. Why? Without editable installation, it is impossible to write correct import 
+statements. `from src.package import module` will not work, because the user has `package` installed, not `src`. And 
+relative imports like `import module` will not work either, because when the tests code (situated in a separate 
+`test` folder) imports the actual code, the relative imports in the actual code fail. This is because relative imports 
+depend on the location of the file that is run, not on the file that contains the import statement. 
+
+The solution is to always have an editable installation of the package under development. Poetry does this 
+automatically when you execute `poetry install`. This makes the package `dsp_tools` importable - just like on a 
+user's machine. And exactly this is the big advantage: With the src layout + editable installation, the setup on the 
+developer's machine is more similar to the user's setup than with any other project layout. 
+
+The concrete advantages of the src layout are:
+
+- import parity
+- The tests run against the package as it will be installed by the user - not against the situation in the 
+  developer's repository.
+- It is obvious to both humans and tools if a folder is a package to be distributed, or not.
+- The editable installation is only able to import files that will also be importable in a regular installation.
+- For the developer, the working directory is the root of the repository, so the root will implicitly be included in 
+  `sys.path`. Users will never have the same current working directory than the developer. So, removing the packages 
+  from the root by putting them into `src` prevents some practices that will not work on the user's machine. 
+
+These things are better explained by the following, often-cited readings:
+
+- [https://blog.ionelmc.ro/2014/05/25/python-packaging](https://blog.ionelmc.ro/2014/05/25/python-packaging)
+- [https://hynek.me/articles/testing-packaging](https://hynek.me/articles/testing-packaging)
+- [https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout)
+
+
+
+### Publishing/distribution
+
+Publishing is automated with GitHub Actions and should _not_ be done manually. Please follow the
+[Pull Request Guidelines](https://docs.dasch.swiss/latest/developers/dsp/contribution/#pull-request-guidelines). If done
+correctly, when merging a pull request into `main`, the `release-please` action will create or update a release 
+PR. This PR will follow semantic versioning and update the change log. Once all desired features are
+merged, the release can be executed by merging this release pull request into `main`. This will trigger actions that
+create a release on GitHub and on PyPI.
+
+Please ensure you have only one pull request per feature.
+
+
+### Publishing manually
+
```

Comment:
This paragraph was moved here, but WITH modifications. Unfortunately, the GitHub Diff is not the best here...